src/promptflow-azure/tests/sdk_cli_azure_test/conftest.py

```diff
@@ -683,15 +687,16 @@ def mocked_run_method(self, *args, **kwargs):
 
         from filelock import FileLock
 
-        count = sum(completed_run_metrics)
+        count = sum(completed_run_metrics.values())
         with FileLock(str(COUNTER_FILE) + ".lock"):
             is_non_zero_file = os.path.isfile(COUNTER_FILE) and os.path.getsize(COUNTER_FILE) > 0
             if is_non_zero_file:
                 with open(COUNTER_FILE, "r", encoding="utf-8") as f:
                     number = json.load(f)
+                    number = number | completed_run_metrics
```

Comment:
why | here?

Reply:
another way of joining two dicts I learnt from stackoverflow, but it doesn't seem to work properly.